src/components/layouts/landing-inner.njk

```diff
@@ -15,8 +15,8 @@
   {% render '@graphic-list' %}
 
   <section class="usa-section">
-    <div class="usa-grid">
-      <h2>Section heading</h2>
+    <div class="grid-container">
+      <h2 class="font-heading-xl">Section heading</h2>
```

Comment:
This shouldn't be necessary if `usa-prose` were working properly

Reply:
Do we expect this section to be used beyond the current design (heading + lead + button), e.g. customized with additional text?

Reply:
Well, I guess I consider this section just an example of "something you could do". It's not really a component, per se, so it's not _really_ important whether it uses `usa-prose` or some other technique. It's certainly OK to leave as is.  

But it does reveal some of what I might consider bugs in `usa-prose`. We'll address them in a subsequent issue, so I'd say it's ok to leave this as-is and move this PR along.